app/assets/stylesheets/application.css.scss

```diff
@@ -110,6 +110,41 @@ body {
   background-position: center center;
 }
 
+.application-body-map_bree{
+  background-image: url('map_bree.png');
+  background-size:     cover;
+  background-repeat:   no-repeat;
+  background-position: center center;
+}
+
+.application-body-map_rivendell{
+  background-image: url('map_rivendell.png');
+  background-size:     cover;
+  background-repeat:   no-repeat;
+  background-position: center center;
+}
+
+.application-body-map_minas-tirith{
```

Comment:
Selector `application-body-map_minas-tirith` should be written in lowercase with hyphens<br>Opening curly brace `{` should be preceded by one space